modules/pruning-images-manual.adoc

```diff
@@ -32,15 +103,17 @@ that have metadata in the cache will not be pushed. Therefore, you must redeploy
 ----
 $ oc rollout restart deployment/image-registry -n openshift-image-registry
 ----
-
-If the integrated registry uses a Redis cache, you must clean the database
-manually.
-
-`oc adm prune images` operations require a route for your registry. Registry
-routes are not created by default.
++
```

Comment:
I assume you fixed the `+` error that currently appears in the netlify preview?

Reply:
I'll double check - thanks!